packages/mdc-list/foundation.ts

```diff
@@ -657,33 +712,41 @@ export class MDCListFoundation extends MDCFoundation<MDCListAdapter> {
    * toggleCheckbox value. User interaction should not toggle list item(s) when
```

Comment:
Looks like we also need to update "toggleCheckbox" name in this comment.

Reply:
fixed 👍